### ISSUE TITLE
Glowroot agent not compatible with JDK 16

### ISSUE THREAD

**Issue by ReillyBrogan** [glowroot/glowroot#788]
```
Unsupported class file major version 60
java.lang.IllegalArgumentException: Unsupported class file major version 60
	at org.glowroot.agent.shaded.org.objectweb.asm.ClassReader.<init>(ClassReader.java:196)
	at org.glowroot.agent.shaded.org.objectweb.asm.ClassReader.<init>(ClassReader.java:177)
	at org.glowroot.agent.shaded.org.objectweb.asm.ClassReader.<init>(ClassReader.java:163)
	at org.glowroot.agent.ManagementFactoryHackClassFileTransformer.transform(ManagementFactoryHackClassFileTransformer.java:48)
	at java.instrument/java.lang.instrument.ClassFileTransformer.transform(Unknown Source)
	at java.instrument/sun.instrument.TransformerManager.transform(Unknown Source)
	at java.instrument/sun.instrument.InstrumentationImpl.transform(Unknown Source)
	at java.base/java.lang.VMAccess.findClassOrNull(Native Method)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at org.glowroot.agent.MainEntryPoint.premain(MainEntryPoint.java:151)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.glowroot.agent.AgentPremain.premain(AgentPremain.java:59)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(Unknown Source)
	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallPremain(Unknown Source)
```
I believe ASM needs to be updated to at least 9.0, but preferably 9.1. Unfortunately I cannot for the life of me get Glowroot to build on my machine, so I'm not able to test this.

@trask Can you please make that change and then put out a new release? Updating ASM should tide us over until JDK 17 at the least.

**Comment by ReillyBrogan** [comment]
Update, I was able to build Glowroot in an Ubuntu docker machine, no idea why it doesn't work on my Linux host machine. I generated an updated agent jar and will test it out tomorrow (but at the very least it didn't exit with the same error when I smoke-tested it).

It was sufficient to update ASM from `7.3.1` to `9.1` (in all the places this is referenced) and then do a global find/replace of `ASM7` -> `ASM9`.
